crates/goose/tests/agent.rs

```diff
@@ -258,6 +258,7 @@ mod tests {
     }
 
     #[tokio::test]
+    #[ignore = "Databricks context truncation tests are flaky - skip in CI"]
```

Comment:
these seem ok - do you want to get rid of these ignores for now @aharvard ?

Reply:
removed and checks are passing. ty!